webapp/certified/views.py

```diff
@@ -331,6 +331,7 @@ def certified_home():
             vendor=vendors,
             query=query,
             offset=offset,
+            limit=limit,
```

Comment:
FYI. This is causing the python test to fail, because now the data source the code is asking for is different, so need to regenerate the cassette.

Reply:
Thank you! fixed it